reefer_monitoring/v1/reefer_v1.0.0-Beta-1.yaml

```diff
@@ -0,0 +1,524 @@
+                        maker: 900
+                        model: N99
+                        operatingState: 1
+                        serialNumber: '123456789'
+                        softwareVersion: 1.1.1
+                      equipmentReference: APZU4812090
+                      equipmentOptions:
+                        - N00
+                        - N05
+                      isConnectedToPowerSource: true
+                      lastDefrost: '2023-10-23T15:40:28Z'
+                      measurements:
+                        ambientTemperature: 12
+                        temperature: 5
+                        temperatureUnit: CEL
+                        o2: 20
+                        co2: 25
+                        relativeHumidity: 98
+                        airExchange: 0
+                        airExchangeUnit: MQH
+                        cargoProbe1Temperature: 6
+                        cargoProbe2Temperature: 6
+                        cargoProbe3Temperature: 6
+                        currentPhaseA: 15
+                        currentPhaseB: 15
+                        currentPhaseC: 15
+                        dischargePressure: 21
+                        dischargeTemperature: 10
+                        evaporatorTemperature: 10
+                        lineFrequency: 60
+                        lineVoltage: 400
+                        returnProbe1Temperature: 10
+                        returnProbe2Temperature: 10
+                        returnTemperature: 10
+                        suctionPressure: 60
+                        suctionTemperature: 10
+                        supplySensor1Temperature: 10
+                        supplySensor2Temperature: 10
+                        supplyTemperature: 10
+                      setpoints:
+                        temperature: -15
+                        temperatureUnit: CEL
+                        o2: 20
+                        co2: 25
+                        humidity: 96
+                      reeferSettings:
+                        controlledAtmosphereMode: 'ON'
+                        defrosting: false
+                        efficiencySetting: '2'
+                      unitModel: N00
+        default:
+          description: Unexpected error
+          headers:
+            API-Version:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/3.0.0#/components/headers/API-Version'
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/ERROR_DOMAIN/2.0.0#/components/schemas/error'
+                  - type: object
+                    properties:
+                      httpMethod:
+                        example: GET
+                  - type: object
+                    properties:
+                      requestUri:
+                        example: 'https://dcsa.org/dcsa/tnt/v1/events'
```

Comment:
I have stopped doing this as codegenerators will not work. It is **not** possible to "overwrite/extend" an attribute like this. Since you are only modifying the example - you can "just" link to the error-object like you have done - and then make an example next to it in an example-attribute (sort of how you did it above with the events-endPoint example

Reply:
Btw - the example in the URL is wrong ;-)

Reply:
cleaned up aligned to other API's use